deploy/role_binding.yaml

```diff
@@ -46,3 +46,16 @@ roleRef:
   kind: Role
   name: remediation-aggregator
   apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRoleBinding
```

Comment:
Does this get used as part of the csv generation? I think you might need to add these capabilities directly into an api-resource-collector role.

Reply:
I intend to break this out into another role (for now was sharing with regular scap scan)